package.json

```diff
@@ -1,9 +1,20 @@
 {
-  "name": "tc39-web-draft",
-  "private": true,
+  "name": "@tc39/tc39.github.io",
```

Comment:
since private is true, i'm not sure why the name matters, but this is fine

Reply:
It was a kind of normalisation.